tests/meson.build

```diff
@@ -0,0 +1,63 @@
+if not expat_dep.found()
+  subdir_done()
+endif
+
+test(
+  'bashTests',
+  find_program('runner.py'),
```

Comment:
I would just find this once and save it as a variable to prevent continually trying to find the same thing from scratch:

```
Program runner.py found: YES (/home/runner/work/exiv2/exiv2/tests/runner.py)
Program runner.py found: YES (/home/runner/work/exiv2/exiv2/tests/runner.py)
Program runner.py found: YES (/home/runner/work/exiv2/exiv2/tests/runner.py)
Program runner.py found: YES (/home/runner/work/exiv2/exiv2/tests/runner.py)
Program runner.py found: YES (/home/runner/work/exiv2/exiv2/tests/runner.py)
Program runner.py found: YES (/home/runner/work/exiv2/exiv2/tests/runner.py)
```

Reply:
Weird. I would think this would be cached.

Reply:
did so. Also switched to a dictionary and foreach loop.